src/index.html

```diff
@@ -4,9 +4,355 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="logoss"
+          class="header__img"
+        >
+      </a>
+
+      <nav class="navigation">
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a href="#apple" class="navigation__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a href="#samsung" class="navigation__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a href="#smartphones" class="navigation__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a
+              href="#laptops"
+              class="navigation__link"
+              data-qa="nav-hover"
+            >
+              Laptops & computers
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a href="#gadgets" class="navigation__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a href="#tablets" class="navigation__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a href="#photo" class="navigation__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a href="#video" class="navigation__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="container__content catalog">
+        <div class="catalog__card card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE iMac 27; Retina 5K Monoblock"
+            class="card__product-img"
+          >
+
+          <h2 class="card__product-title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+          <p class="card__product-code">Product code: 195434</p>
```

Comment:
IHO, is a little better because we can see the content without reading the whole line

```suggestion
          <p class="card__product-code">
            Product code: 195434
          </p>
```